tools/export.py

```diff
@@ -223,31 +212,30 @@ def main(args):
         cfg.load_from = args.checkpoint
         cfg.resume_from = None
         compression_ctrl, model = wrap_nncf_model(model, cfg, None, get_fake_input)
-        compression_ctrl.prepare_for_export()
+        # TODO: apply the following string for NNCF 1.5.*
+        # compression_ctrl.prepare_for_export()
```

Comment:
This should not be part of this PR. Seems this is the result of a target branch merge. A bit confused if [7619c5a](https://github.com/openvinotoolkit/mmdetection/pull/84/commits/7619c5a7c56d4fd0eb73afb1e78534d968206c08) is a merge commit or not. It should be a merge.